mcache/README.md

```diff
@@ -2,36 +2,43 @@
 
 ## Overview
 
-The Agent's memcache check lets you track memcache's memory use, hits, misses, evictions, fill percent, and much more.
+The Agent's Memcache check lets you track Memcache's memory use, hits, misses, evictions, fill percent, and much more.
 
 ## Setup
-
-Follow the instructions below to install and configure this check for an Agent running on a host. For containerized environments, see the [Autodiscovery Integration Templates][1] for guidance on applying these instructions.
-
 ### Installation
 
-The memcache check is included in the [Datadog Agent][2] package, so you don't need to install anything else on your memcache servers.
+The Memcache check is included in the [Datadog Agent][2] package, so you don't need to install anything else on your Memcache servers.
 
 ### Configuration
 
+Follow the instructions below to configure this check for an Agent running on a host. For containerized environments, see the [Containerized](#containerized) section
+
+#### Host
+
 1. Edit the `mcache.d/conf.yaml` file, in the `conf.d/` folder at the root of your [Agent's configuration directory][3].
   See the [sample mcache.d/conf.yaml][4] for all available configuration options:
 
     ```yaml
       init_config:
 
       instances:
-        - url: localhost  # url used to connect to the memcached instance
-          port: 11212 # optional; default is 11211
-      #    socket: /path/to/memcache/socket # alternative to url/port; 'dd-agent' user must have read/write permission
-          options:
-            items: false # set to true to collect items stats
-            slabs: false # set to true to collect slabs stats
-      #    tags:
-      #    - optional_tag
+        ## @param url - string - required
+        ## url used to connect to the Memcached instance.
+        #
+        - url: localhost
     ```
 
-2. [Restart the Agent][5] to begin sending memcache metrics to Datadog.
+2. [Restart the Agent][5] to begin sending Memcache metrics to Datadog.
+
+#### Containerized
```

Comment:
for integrations like this one that have an auto_conf template that will automatically configure the integration even if the user doesn't provide a config, do you think it's worth adding a special note to let people know that's a thing? See https://github.com/DataDog/integrations-core/blob/master/mcache/datadog_checks/mcache/data/auto_conf.yaml

Reply:
We have no public documentation about auto configuration :/ 
I think as a first step let's not advocate for the auto-conf. Since it's automated, there is some kind of "magic", but i fear that if we document it people will ask why not all integrations have auto-conf and it might generate frictions over the line of: "it used to work, but now it's not working anymore after i pushed a change to my techno".

When all compatible integrations have their AD documented, let's see how to explain auto-conf in a factorised way, what do you think?

Reply:
So I agree we should have a centralized page to explain auto-conf, but I also think all the integrations that have it should mention a short note about it in this section.
If your proposal is to wait for having this page to add a note to each integration and point to that page, that's cool with me 👌

Reply:
Card created.